libs/core/src/lib/components/wrapper-component.ts

```diff
@@ -118,7 +118,7 @@ export abstract class ReactWrapperComponent<TProps extends {}> implements AfterV
     if (isReactNode(this.reactNodeRef.nativeElement)) {
       const stringValue = typeof value === 'string' ? value : stylenames(value);
       this.reactNodeRef.nativeElement.setProperty('style', toStyle(stringValue));
-      this.changeDetectorRef.detectChanges();
+      this.markForCheck();
```

Comment:
Did you encounter any issues that warrant this change? `markForCheck()` does more than `this.changeDetectorRef.detectChanges()`.

Reply:
See other reply on similar change